docs/current_docs/configuration/cache.mdx

```diff
@@ -0,0 +1,37 @@
+---
+slug: /configuration/cache
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Cache
+
+Dagger caches two types of data:
+
+1. Layers: This refers to build instructions and the results of some API calls. This cache is implemented by Buildkit.
+2. Volumes: This refers to the contents of a Dagger filesystem volume and is persisted across Dagger Engine sessions. It is implemented by Dagger (distinct from Buildkit).
+
+## Cache inspection
+
+- Show all the cache entry metadata:
+
+    ```shell
+    echo '{daggerEngine{localCache{entrySet{entries{description diskSpaceBytes}}}}}' | dagger query
+    ```
+
+- Show high level summaries of cache usage:
+
+    ```shell
+    echo '{daggerEngine{localCache{entrySet{entryCount diskSpaceBytes}}}}' | dagger query
+    ```
+
+## Cache pruning
+
+The layer cache size configuration is managed by [Buildkit's garbage collector](https://github.com/moby/buildkit/blob/master/docs/buildkitd.toml.md?plain=1#L83-L94).
```

Comment:
```diff
- Buildkit
+ BuildKit
```

Reply:
I would pin this link to a specific version - `master` is guaranteed to change & this will break. I believe that it did since this PR was opened:

<img width="1552" alt="image" src="https://github.com/user-attachments/assets/bd89204d-69b1-4038-bdbf-d5e578d1ad53">

Reply:
Updated